gravityflow/gravityflow
# Fix assignee reassignment on Gravity Forms 2.5

## Description
[Issue# 44](https://github.com/gravityflow/backlog/issues/44)

## Testing instructions

1. Create a workflow with Approval step assigned to user-1.
2. Process an entry for the workflow.
3. Check user-1's workflow for this entry but don't approve the workflow step.
4. Update workflow settings, reassign the Approval assignee to user-2.
5. With Gravity Forms 2.5, the workflow entry would still display in user-1, and not on user-2.
6. With Gravity Forms 2.4, the workflow entry would update to user-2.
7. Switch to this branch. Check with Gravity Forms 2.5, the workflow entry does update to the user-2 when workflow step settings are changed.
8. Likewise confirm with Gravity Forms 2.4, the workflow entry does update to the user-2 when workflow step settings are changed.

## Automated Test Enhancements
N/A
 
## Screenshots
N/A

## Documentation Changes?
N/A

## Checklist:
